school/decorator.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'nameable'
+class Decorator < NameAble
+  def initialize(nameable)
```

Comment:
please kindly add an ` attr_accessor: nameable` for the initialization of the decorator 😄

Reply:
thank you for valuable feedback 💯